examples/advanced/cyclic/job/app/config/config_fed_client.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Not sure we want to publish this example since it is not a production quality one. We don't want users to use this example and then come back with all kinds of questions. 

We will need to develop cyclic and swarm learning examples based on the prod quality impl.

Reply:
removed the example.